server/src/test/java/io/unitycatalog/server/base/catalog/BaseCatalogCRUDTest.java

```diff
@@ -1,39 +1,39 @@
 package io.unitycatalog.server.base.catalog;
 
 import static io.unitycatalog.server.utils.TestUtils.*;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.tuple;
 
 import io.unitycatalog.client.ApiException;
 import io.unitycatalog.client.model.CatalogInfo;
 import io.unitycatalog.client.model.CreateCatalog;
 import io.unitycatalog.client.model.UpdateCatalog;
 import io.unitycatalog.server.base.BaseCRUDTest;
 import java.util.List;
-import java.util.Objects;
 import java.util.Optional;
 import org.junit.jupiter.api.Test;
 
 public abstract class BaseCatalogCRUDTest extends BaseCRUDTest {
 
   protected void assertCatalog(CatalogInfo catalogInfo, String name, String comment) {
-    assertEquals(name, catalogInfo.getName());
-    assertEquals(comment, catalogInfo.getComment());
-    assertNotNull(catalogInfo.getCreatedAt());
+    assertThat(catalogInfo.getName()).isEqualTo(name);
+    assertThat(catalogInfo.getComment()).isEqualTo(comment);
+    assertThat(catalogInfo.getCreatedAt()).isNotNull();
     // TODO: Also assert properties once CLI supports it
   }
 
   protected void assertCatalogExists(List<CatalogInfo> catalogList, String name, String comment) {
-    assertTrue(
-        catalogList.stream()
-            .anyMatch(
-                c -> Objects.equals(c.getName(), name) && Objects.equals(c.getComment(), comment)));
+    assertThat(catalogList)
+        .extracting(CatalogInfo::getName, CatalogInfo::getComment)
+        .as("Catalog list should contain a catalog with name '%s' and comment '%s'", name, comment)
+        .containsAnyOf(tuple(name, comment));
   }
 
   protected void assertCatalogNotExists(List<CatalogInfo> catalogList, String name) {
-    assertFalse(catalogList.stream().anyMatch(c -> Objects.equals(c.getName(), name)));
+    assertThat(catalogList)
```

Comment:
this can be simplified to `assertThat(catalogList).anyMatch(c -> Objects.equals(c.getName(), name))`